08-Secure by default/Secure Templates/ARM/VirtualMachine-Linux.json

```diff
@@ -145,6 +145,12 @@
       "metadata": {
         "description": "If “Auto-provisioning” for AMA is turned on in Azure Defender configuration, you can skip installation of AMA agent during VM creation as MDC will auto deploy AMA agent with desired configuations."
       }
+    },
+    "deployAzureAADAuthExtension": {
```

Comment:
Are these really secure by default since we are asking user for input instead of setting it ourselves?